addons/medical_engine/functions/fnc_handleDamage.sqf

```diff
@@ -32,9 +32,14 @@ if (_hitPoint isEqualTo "") then {
 if !(isDamageAllowed _unit && {_unit getVariable [QEGVAR(medical,allowDamage), true]}) exitWith {_oldDamage};
 
 private _newDamage = _damage - _oldDamage;
-// Get armor value of hitpoint and calculate damage before armor
-private _armor = [_unit, _hitpoint] call FUNC(getHitpointArmor);
+// Get scaled armor value of hitpoint and calculate damage before armor
+// We scale using passThrough to handle explosive-resistant armor properly (#9063)
+// We need realDamage to determine which limb was hit correctly
+[_unit, _hitpoint] call FUNC(getHitpointArmor) params ["_armor", "_armorScaled"];
 private _realDamage = _newDamage * _armor;
+if (_hitPoint isNotEqualTo "#structural") then {
+    _newDamage = _newDamage * (_armor/_armorScaled);
```

Comment:
I think we could make this scaling a setting?
just a 0-1 lerp between old way and new

Reply:
Why?